src/modules/netinstall/NetInstallPage.cpp

```diff
@@ -62,4 +63,21 @@ void
 NetInstallPage::onActivate()
 {
     ui->groupswidget->setFocus();
+
+    // The NetInstallSelect global sotrage value can be used to make additional items selected by default
+    Calamares::GlobalStorage* gs = Calamares::JobQueue::instance()->globalStorage();
+    if ( gs->contains( "NetinstallSelect" ) && gs->value( "NetinstallSelect" ).canConvert( QVariant::StringList ) )
```

Comment:
This could probably be shorter by just *doing* the conversion to stringlist, and then checking the list if it is empty?